medicines/web/infrastructure/README.md

```diff
@@ -4,21 +4,62 @@ Instructions on getting the website set up.
 As steps are completed, the developer completing those steps is responsible for updating this document.
 This document should become a guide for setting up the website infrastructure on a new environment and for connecting to the existing environment.
 
-## To-do Steps
+## Docker
 
-This infrastructure is a work-in-progress.
-The steps below are a guide to get to the desired infrastructure.
+If you want to use Docker to run the website locally, the good news is that you can!
+
+```sh
+docker-compose up
+```
+
+It's as simple as that.
+
+### Building, pushing, running from registry
+
+Provisioning the website as a docker image, ready for use in Azure Kubernetes cluster (see [_To-do Steps_](#to-do-steps)), is a little more complicated that running it locally. For all of the below steps, you'll want to be in `/medicines/web`.
+
+1. Build
+
+   ```sh
+   docker-compose build
+   ```
+   
+   This will build the docker image locally, using the `docker-compose.yml`.
+   It runs the `Dockerfile`, and tags the built image as `web:latest`.
 
-1. Docker time!
+2. Login
 
-   Create a `Dockerfile` in `/medicines/web` which will run `yarn dev`, serving up the Nextjs website.
-   Verify that this is working locally by running the docker image, and connecting to the website.
+   ```sh
+   az acr login --name mhraproductsdevregistry
+   ```
+   
+   This should work, provided you're logged into the right Azure account. 
+
+3. Tag and push
+
+   ```sh
+   docker tag web mhraproductsdevregistry.azurecr.io/products/web
+   docker push mhraproductsdevregistry.azurecr.io/products/web
+   ```
+   
+   The first line here lets docker know that the origin for the pushed image should be your local `web` image.
+   The second line does the pushing.
+   
+All being well, you'll now be able to run the pushed image, locally. Test it out:
 
-   You will need to push the docker image to the container registry for each of our environments.
+```sh
+docker run -it --rm -p 3000:3000 mhraproductsdevregistry.azurecr.io/products/web
```

Comment:
Why `--rm` ?

Reply:
I had copied this from a guide without questioning, thanks for highlighting. The docs are pretty clear, [it's all about cleaning up the container afterwards][Docker documentation].

> By default a container’s file system persists even after the container exits. This makes debugging a lot easier (since you can inspect the final state) and you retain all your data by default. But if you are running short-term foreground processes, these container file systems can really pile up. If instead you’d like Docker to automatically clean up the container and remove the file system when the container exits, you can add the `--rm` flag.

[Docker documentation]: https://docs.docker.com/engine/reference/run/#clean-up---rm "Clean up | Docker Documentation"